example/page-base.html

```diff
@@ -0,0 +1,98 @@
+<head>
+  <title>Page title</title>
+  <meta charset="utf-8" />
+  <meta name="description" content="" />
+  <!-- Place here the head part: https://web2018.epfl.ch/0.0.1/includes/head.html -->
+</head>
+<body>
+
+  <!-- Header - Example: header light -->
+  <header role="banner" class="header header-light">
+    <div class="drawer mr-3 mr-xl-0">
+      <button class="drawer-toggle">
+        <svg aria-hidden="true" class="icon">
+          <use xlink:href="#icon-chevron-right"></use>
+        </svg>
+      </button>
+      <a href="//www.epfl.ch" class="drawer-link" style="">
+        <span class="text">
+          Go to main site
+        </span>
+      </a>
+    </div>
+
+    <div class="header-light-content">
+      <a href="//www.epfl.ch" class="logo">
+        <img src="https://web2018.epfl.ch/0.0.1/icons/epfl-logo.svg" alt="Logo EPFL, École polytechnique fédérale de Lausanne" class="img-fluid">
+      </a>
+
+      <h1>Example Home</h1>
+      <ul aria-hidden="true" class="nav-header d-none d-xl-flex"></ul>
+      <nav aria-label="Change the language" class="nav-lang">
+        <ul>
+          <li>
+            <a href="/index.html" aria-label="English" class="active">EN</a>
+          </li>
+          <li>
+            <a href="/index.fr.html" aria-label="Français" class="">FR</a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+
+    <form action="//search.epfl.ch" class="d-lg-none">
+      <div class="input-group search" role="search">
+        <div class="input-group-prepend">
+          <span class="input-group-text">
+            <svg class="icon" aria-hidden="true">
+              <use xlink:href="#icon-search"></use>
+            </svg>
+          </span>
+        </div>
+        <label for="search" class="sr-only">Search</label>
+        <input type="text" class="form-control" name="q" id="search" placeholder="Search">
+      </div>
+    </form>
+
+    <div class="btn btn-secondary nav-toggle-mobile d-xl-none">
+      Menu
+      <div class="hamburger">
+        <span></span>
+        <span></span>
+        <span></span>
+      </div>
+    </div>
+  </header>
+
+  <div class="main-container">
+
+    <!-- Breadcrumb - Example -->
+    <div class="breadcrumb-container">
+      <nav aria-label="breadcrumb" class="breadcrumb-wrapper" id="breadcrumb-wrapper">
+        <ol class="breadcrumb">
+          <li class="breadcrumb-item">
+            <a href="//www.epfl.ch" title="Home" aria-label="Home">
+              <svg class="icon" aria-hidden="true"><use xlink:href="#icon-home"></use></svg>
+            </a>
+          </li>
+          <li class="breadcrumb-item">
+            <a href="/index.html">Example Home</a>
+          </li>
+        </ol>
+      </nav>
+    </div>
+
+    <!-- Main... -->
+
+  </div>
+
+  <!-- Footer -->
+  <div class="bg-gray-100 pt-5 pb-3">
+    <div class="container">
+      <!-- Place here the footer: https://web2018.epfl.ch/0.0.1/includes/footer-scripts.html -->
+    </div>
+  </div>
+</body>
```

Comment:
Here: <!-- Place here the footer scripts: https://web2018.epfl.ch/0.0.1/includes/footer-scripts.html -->